CONTRIBUTING.md

```diff
@@ -1,55 +1,260 @@
+
+By filling out the "_Lorem ipsum_" sections of the pull request template with as much detail as possible, you will make it really easy for someone to review your contribution!
+
+If you have opened the pull request early and know that its contents are not ready for review or to be merged, add "[WIP]" at the start of the pull request title, which stands for "Work in Progress".
+When you are happy with it and are happy for it to be merged into the main repository, change the "[WIP]" in the title of the pull request to "[Ready for review]".
+
+A member of _EDS book_ team will then review your changes to confirm that they can be merged into the main repository.
+A [review][github-review] will probably consist of a few questions to help clarify the work you've done.
+Keep an eye on your GitHub notifications and be prepared to join in that conversation.
+
+You can update your [fork][github-fork] of _EDS book_ [repository][eds-book-repo] and the pull request will automatically update with those changes.
+You don't need to submit a new pull request when you make a change in response to a review.
+
+You can also submit pull requests to other contributors' branches!
+Do you see an [open pull request](https://github.com/alan-turing-institute/environmental-ds-book/pulls) that you find interesting and want to contribute to?
+Simply make your edits on their files and open a pull request to their branch!
+
+What happens if the continuous integration (CI) fails (for example, if the pull request notifies you that "Some checks were not successful")?
+The CI could fail for a number of reasons.
+At the bottom of the pull request, where it says whether your build passed or failed, you can click “Details” next to the test, which takes you to a CI run log site.
+If you have the write access to the repo, you can view the log or rerun the checks by clicking the “Restart build” button in the top right.
+You can learn more about CI in the [Continuous Integration chapter](https://the-turing-way.netlify.app/reproducible-research/ci.html)!
+
+GitHub has a [nice introduction][github-flow] to the pull request workflow, but please [get in touch](#get-in-touch) if you have any questions or ideas :bulb:.
+
+## Local development
+
+You can build and host the book website locally. The steps are:
+
+### To build book locally
+
+1. Install the required software to build the book, optionally in a [virtual environment](https://packaging.python.org/tutorials/installing-packages/#creating-virtual-environments):
+
+   ```
+   pip install -r book/requirements.txt
+   ```
+
+2. You can now build or refresh the book using:
+
+   ```
+   cd ./book/website
+   jupyter-book build .
+   ```
+
+## Representing _EDS book_
+
+We would LOVE people to give talks about the project or represent _The Turing Way_ in other ways!
+We have created a [promotion pack](https://github.com/alan-turing-institute/environmental-ds-book/tree/master/communications/promotion-pack) for you to reuse.
```

Comment:
Do we mean _The Turing Way_ or _The EDS Book_ ?

Reply:
You're right. I should be _EDS book_

Reply:
Fixed in commit [659ba3](https://github.com/alan-turing-institute/environmental-ds-book/pull/219/commits/659ba3b4c94daa3e007095c0bca8c30a6189f2f6)